api/net/TNO.API.csproj

```diff
@@ -14,6 +14,7 @@
     <PackageReference Include="Microsoft.AspNetCore.Mvc.Versioning.ApiExplorer" Version="5.0.0" />
     <PackageReference Include="Swashbuckle.AspNetCore.Annotations" Version="6.3.1" />
     <PackageReference Include="MimeTypeMapOfficial" Version="1.0.17"/>
+    <PackageReference Include="Microsoft.CognitiveServices.Speech" Version="1.23.0" />
```

Comment:
Why does the API need this dependency?

Reply:
The documentation said this was how you import the functionality needed for:

using Microsoft.CognitiveServices.Speech;
using Microsoft.CognitiveServices.Speech.Audio;

which is where classes like: AudioStreamFormat, PushAudioInputStream and SpeechRecognizer are located.

Reply:
But the API is not performing this functionality, the Transcription Service is.

Reply:
I added this because I initially implemented transcription as an API endpoint. I've removed this line.